packages/fast-components-react-base/src/context-menu-item/README.md

```diff
@@ -0,0 +1,2 @@
+# Context menu item
+An menu item component designed to work as a child of the *context-menu*.
```

Comment:
You switch between hyphenating and not with context menu. I think all occurrences should **not** be hyphenated.